tree/tree/inc/TTree.h

```diff
@@ -160,6 +163,11 @@ class TTree : public TNamed, public TAttLine, public TAttFill, public TAttMarker
    TTree(const TTree& tt);              // not implemented
    TTree& operator=(const TTree& tt);   // not implemented
 
+#ifdef R__USE_IMT
+   mutable Bool_t fIMTFlush{false};               ///<! True if we are doing a multithreaded flush.
```

Comment:
Fair enough, a comment here or at the use place along the line of this answer ought to be added.